### PR TITLE
Update a few dependencies

### PR DESCRIPTION
**Pull request by pahill**
Ktor
Coroutines
AGP
Kotlin

### CODE REVIEW

File: gradle/libs.versions.toml [hunk 1]
```diff
@@ -1,14 +1,14 @@
 [versions]
-plugin-android = "8.0.2"
-plugin-kotlin = "1.8.21"
+plugin-android = "8.1.0-rc01"
```
Comment (terrakok) [inline gradle/libs.versions.toml:2]: do we want to use not stable versions?

Comment (terrakok) [reply gradle/libs.versions.toml:2]: is it necessary?

Comment (pahill) [reply gradle/libs.versions.toml:2]: It was necessary yes, there were a couple of things that needed to be upgraded together. We can however wait until the Android plugin is stable?

Comment (pahill) [reply gradle/libs.versions.toml]: @terrakok You can re-review this comment, it's been updated

**Pull request merged**